plugins/techdocs-react/api-report.md

```diff
@@ -0,0 +1,116 @@
+## API Report File for "@backstage/plugin-techdocs-react"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { AsyncState } from 'react-use/lib/useAsync';
+import { ComponentType } from 'react';
+import { CompoundEntityRef } from '@backstage/catalog-model';
+import { Context } from 'react';
+import { Dispatch } from 'react';
+import { Entity } from '@backstage/catalog-model';
+import { Extension } from '@backstage/core-plugin-api';
+import { default as React_2 } from 'react';
+import { SetStateAction } from 'react';
+import { VersionedValue } from '@backstage/version-bridge';
+
+// @alpha
+export function createTechDocsAddon<TComponentProps>(
+  options: TechDocsAddonOptions<TComponentProps>,
+): Extension<ComponentType<TComponentProps>>;
+
+// @alpha (undocumented)
+export const defaultTechDocsReaderPageValue: TechDocsReaderPageValue;
+
+// @alpha
+export const TECHDOCS_ADDONS_WRAPPER_KEY = 'techdocs.addons.wrapper.v1';
+
+// @alpha
+export enum TechDocsAddonLocations {
```

Comment:
This is a hot topic, but there's an almost-consensus it seems, to avoid TS `enum`. Could make this just an `Object.freeze({ CONTENT: ... }` instead